project/Build.scala

```diff
@@ -1130,6 +1130,7 @@ object Build {
     enablePlugins(DottyJSPlugin).
     dependsOn(`scala3-library-bootstrappedJS`).
     settings(
+      bspEnabled := false,
```

Comment:
Maybe we should add a small note here then to let Metals users know that if they want Metals support in Dotty they need to use sbt as their build server and uncomment this.

Reply:
Added above, since this one should not be uncommented (it causes infinite compilation)

Reply:
Ah sorry, yes I meant the other one.